complaint/tests.py

```diff
@@ -49,6 +50,6 @@ class URLTest(TestCase):
 
     def test_complaint_urls(self):
         for url_name in self.url_names:
-            response = client.get(reverse("complaints:{}".format(url_name)))
+            response = client.get(reverse("{}".format(url_name)))
```

Comment:
If you're going to remove the namespace here, can't this just be simplified to `reverse(url_name)`?